docs/sql-guide/statements/statement-insert-bulk.md

```diff
@@ -0,0 +1,165 @@
+---
+title: BULK INSERT
+layout: default
+parent: Statements
+grand_parent: SQL guide
+---
+
+# BULK INSERT statement
+
+BULK INSERT data from CSV, NDJSON, URL or streaming sources to a FeatureBase table.
+
+## BNF diagrams
+
+![expr](/assets/images/sql-guide/bulk_insert_stmt.svg)
+
+### Column list
+![expr](/assets/images/sql-guide/column_list.svg)
+
+### Map list
+![expr](/assets/images/sql-guide/map_list.svg)
+
+### Transform clause
+![expr](/assets/images/sql-guide/map_list.svg)
+
+### With clause
+![expr](/assets/images/sql-guide/bulk_insert_options.svg)
+![expr](/assets/images/sql-guide/bulk_insert_option.svg)
+
+## DDL syntax
+
+```
+BULK [INSERT | REPLACE]
```

Comment:
get rid of replace. not supported right now

Reply:
Done